### ISSUE TITLE
Old existing  working page `data` isn't parsed correctly in v4.0.4; works fine with v4.0.2

### ISSUE THREAD

**Issue by Gompje** [outl1ne/nova-page-manager#103]
Composer update bumped this package from 4.0.2 →  4.0.4. This broke my pages. The existing, not changed data wasn't correctly mapped anymore. I reverted back to 4.0.2 and everything works as expected.

It threw me off given that I did not expect a small version bump to break things ;-)

I did not fully debug this, it was easier to revert. but it seems to be in `nova_resolve_fields_data` function and line 303-ish
```
((isset($value->component)) && ($value->component === 'panel') && (nova_page_manager_sanitize_panel_name($value->name) === $fieldAttribute))
```
that last comparison is failing. The keys are however the same at first glance, but like I said I did not dig in deep. 
v4.0.2 works perfectly.

**Comment by Tarpsvo** [comment]
Ah crap, we did change the panel name slugification logic. The `nova_page_manager_sanitize_panel_name` function apparently now handles the panel name differently.

If I were you, I would fix the nova-page-manager to version `4.0.3` and just use that version. I won't be making any 4.X releases anymore for nova-page-manager (unless absolutely necessary). The next version will be a complete rewrite of the package with better localization logic.

**Comment by matthewjumpsoffbuildings** [comment]
@Tarpsvo this seems completely broken - see #134
